Basic/DistributedAuthoritySocialHub/Assets/Scripts/Services/VivoxManager.cs

```diff
@@ -9,14 +11,29 @@ namespace Unity.Multiplayer.Samples.SocialHub.Services
 {
     class VivoxManager : MonoBehaviour
     {
-        internal static VivoxManager Instance { get; set; }
-        string ChannelName { get; set; }
+        string m_TextChannelName;
+        string m_VoiceChannelName;
+
+        internal static VivoxManager Instance { get; private set; }
+
+        PlayersTopUIController m_PlayersTopUIController;
+        PlayersTopUIController PlayersTopUIController
```

Comment:
This doesn't seem like it belongs here, and could be done inside the game scene. VivoxManager exists for the duration of the application, so probably game components should listen for these Vivox events, and modify UI.